Transmission.xcodeproj/project.pbxproj

```diff
@@ -7,37 +7,37 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		0A6169A70FE5C9A200C66CE6 /* bitfield.c in Sources */ = {isa = PBXBuildFile; fileRef = 0A6169A50FE5C9A200C66CE6 /* bitfield.c */; };
+		0A6169A70FE5C9A200C66CE6 /* bitfield.c in Sources */ = {isa = PBXBuildFile; fileRef = 0A6169A50FE5C9A200C66CE6 /* bitfield.c */; settings = {COMPILER_FLAGS = "-fno-objc-arc"; }; };
```

Comment:
It looks like it was added by the automatic converter when I explicitly marked that all the C libraries are not to be converted.